IdentityCore/src/cache/mac/MSIDMacACLKeychainAccessor.m

```diff
@@ -216,7 +233,7 @@ - (BOOL)saveData:(NSData *)data
     updateQuery[(id)kSecValueData] = data;
     
     __block OSStatus status;
-    dispatch_barrier_sync(s_synchronizationQueue, ^{
+    dispatch_barrier_sync(MSIDMacACLKeychainAccessor.s_synchronizationQueue, ^{
```

Comment:
nit: `self.class` instead of `MSIDMacACLKeychainAccessor`. if we rename the class name in future, we won't need to update here.